Proposed/asc048.md

```diff
@@ -0,0 +1,52 @@
+#### **Title**
+New instruction: `image`
+
+#### **Authorship**
+Asuka Ota <asuka.ota@strateos.com>
+
+Varun Kanwar <varunkanwar1@gmail.com>
+
+#### **Motivation**
+Imaging during the course of chemical and bioscience experiments provides valuable information about samples, such as opacity, color, dryness, crystal structure, and indicates microbial growth and the presence of contaminants. It is also useful as a tool to flag anomalies, and detect unexpected or unintended reactions. The currently-existing `image_plate` instruction offers limited support for these needs, and enables only top-down imagery of SBS-format plates. This ASC aims to support imaging of samples in containers with disparate shapes and well-formats, from several angles, and to provide some control over lighting conditions and magnification.
+
+#### **Proposal**
+We propose a versatile `image` instruction to allow for visible-light imaging operations on a broad range of container types.
+
+```
+{
+  “op”: “image”,
+  /* A plate, tube, vial, or bottle */
+  “object”: Container,
+  /* Angle of image capture */
+  “mode”: Enum(“top”, ”bottom”, “side”),
+  /* Number of images taken of the container. Optional; should default to 1 */
+  “num_images”: Int
+  /* Image data output */
+  “dataref”: Data,
```

Comment:
Update docstring and key to "dataref": Str
This field should reference the name of the dataset, not the returned data object.